genesyscloud/resource_exporter_custom.go

```diff
@@ -0,0 +1,37 @@
+package genesyscloud
+
+import (
+	"fmt"
+)
+
+/*
+The resource_genesyscloud_routing_queue object has the concept of bullseye ring with a member_groups attribute.
+The routing team has overloaded the meaning of the member_groups so you can id and then define what "type" of id this is.
+This causes problems with the exporter because our export process expects id to map to a specific resource.
+
+This customer custom router will look at the member_group_type and resolve whether it is SKILLGROUP, GROUP type.  It will then
+find the appropriate resource out of the exporters and build a reference appropriately.
+*/
+func MemberGroupsResolver(configMap map[string]interface{}, exporters map[string]*ResourceExporter) error {
+
+	var exporter *ResourceExporter
+	memberGroupType := configMap["member_group_type"]
+	memberGroupID := configMap["member_group_id"].(string)
+
+	switch memberGroupType {
+	case "SKILLGROUP":
+		exporter = exporters["genesyscloud_routing_skill_group"]
```

Comment:
There needs to be some check here to see if this key exists in the map. If a user forgets to include `genesyscloud_routing_skill_group` or `genesyscloud_group` in the `resource_types` array when they're exporting, they'll see a panic nil pointer exception